src/Microsoft.Azure.SignalR.Common/Endpoints/AccessKey.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Microsoft.Azure.SignalR
+{
+    internal class AccessKey
+    {
+        public string Value { get; set; }
+        public string Id { get; set; }
+
+        public AccessKey(string key, string keyId = "signalr")
+        {
+            Value = key;
+            Id = keyId;

```

Comment:
use key.GetHashCode as the default value of `Id` to avoid breaking change? keyId is used as the key of the cache for `SignatureProvider` so a const value is not correct for multiple endpoint case